alerter.c

```diff
@@ -13,7 +16,15 @@ int networkAlertStub(float celcius) {
 
 void alertInCelcius(float farenheit) {
     float celcius = (farenheit - 32) * 5 / 9;
-    int returnCode = networkAlertStub(celcius);
+    int returnCode;
+    if(Current_Env == INTEGRATION_ENV)

```

Comment:
Using conditional code to execute test or production code increases complexity as the code size grows. Maintenance becomes difficult.
Test code and production code should be maintained separately.

Production code should not be aware of test environment or production envt.

Use dependency injection to solve this problem.